charts/policies/user-group-psp/values.yaml

```diff
@@ -0,0 +1,29 @@
+---
+kind: ClusterAdmissionPolicy
+metadata:
+  name: user-group-psp
+  namespace: default # used when kind is AdmissionPolicy
+spec:
+  module:
+    repository: "registry://ghcr.io/kubewarden/policies/user-group-psp"
+    tag: "v0.2.0"
+  mode: "protect"
+  mutating: true
+  rules:
+    -
+      apiGroups:
+        - ""
+      apiVersions:
+        - v1
+      resources:
+        - pods
+      operations:
+        - CREATE
+        - UPDATE
+  settings:
+    run_as_user:
+      rule: "RunAsAny"
+    run_as_group:
+      rule: "RunAsAny"
+    supplemental_groups:
+      rule: "RunAsAny"
```

Comment:
added the minimum amount of settings in `values.yaml` needed, so they can be changed via questions. Notice that the `templates/policy.yaml` also contains the minimum amount of logic needed to render them correctly.